src/Frontend/src/views/throughputreport/endpoints/DetectedListView.vue

```diff
@@ -0,0 +1,277 @@
+}
+
+function proceedWithChangesWarning() {
+  showBulkUpdateWarning.value = false;
+  updateIndicators();
+}
+
+async function loadData() {
+  const results = await throughputClient.endpoints();
+
+  data.value = results.filter((row) => row.is_known_endpoint === (props.source === DataSource.wellKnownEndpoint));
+}
+
+function nameFilterChanged(event: Event) {
+  filterData.name = (event.target as HTMLInputElement).value;
+}
+
+function sortChanged(item: Item) {
+  filterData.sort = item.value;
+}
+
+function searchTypeChanged(event: Event) {
+  filterData.nameFilterType = (event.target as HTMLInputElement).value as NameFilterType;
+}
+
+function updateIndicator(event: Event, name: string) {
+  const value = (event.target as HTMLSelectElement).value;
+  updateDataChanged(name, (item) => (item.indicator = value));
+  save();
+}
+
+function showBulkUpdateIndicatorWarning(indicator: UserIndicator) {
+  bulkOperation.value = indicator;
+  showBulkUpdateWarning.value = true;
+}
+
+function updateIndicators() {
+  filteredData.value.forEach((item) => {
+    updateDataChanged(item.name, (item) => (item.indicator = bulkOperation.value));
+  });
+  save();
+}
+
+function updateDataChanged(name: string, action: (item: { indicator: string }) => void) {
+  const item = dataChanges.value.get(name);
+  if (item) {
+    action(item);
+  }
+}
+
+function showUnsetChanged(event: Event) {
+  filterData.showUnsetOnly = (event.target as HTMLInputElement).checked;
+}
+
+const bulkOperation = ref<UserIndicator>(UserIndicator.NServiceBusEndpoint);
+
+function getDefaultEndpointType(row: EndpointThroughputSummary) {
+  if (row.is_known_endpoint) {
+    return UserIndicator.NServiceBusEndpoint.toString();
+  }
+
+  return undefined;
+}
+
+async function save() {
+  const updateData: UpdateUserIndicator[] = [];
+  dataChanges.value.forEach((value, key) => {
+    updateData.push({ name: key, user_indicator: value.indicator });
+  });
+
+  await throughputClient.updateIndicators(updateData);
+
+  useShowToast(TYPE.SUCCESS, "Saved", "", false, { timeout: 1000 });
+
+  await loadData();
+}
+</script>
+
+<template>
+  <div>
+    <div class="row filters">
+      <div class="col">
+        <div class="text-search-container">
+          <div>
+            <select class="form-select text-search format-text" @change="searchTypeChanged">
+              <option v-for="item in filterNameOptions" :value="item.text" :key="item.text">{{ item.text }}</option>
+            </select>
+          </div>
+          <div>
+            <input type="search" class="form-control format-text" :value="filterData.name" @input="nameFilterChanged" placeholder="Filter by name..." />
+          </div>
+        </div>
+      </div>
+      <div class="col" style="align-content: center">
+        <div>
+          <input type="checkbox" class="check-label" id="showUnsetOnly" @input="showUnsetChanged" />
+          <label for="showUnsetOnly">Show only not set endpoint types</label>
```

Comment:
I think we use capital letters for these else where?

```suggestion
          <label for="showUnsetOnly">Show only not set Endpoint Types</label>
```